test/client/getPaths.ts

```diff
@@ -1,7 +1,7 @@
 import addresses from "../fixtures/addresses.json";
 import requests from "../fixtures/requests";
 import rippled from "../fixtures/rippled";
-import setupClient from "../setupClient";
+import { setupClient, teardownClient } from "../setupClient";
 import { assertRejects } from "../testUtils";
 // import responses from '../fixtures/responses'
```

Comment:
let's get rid of this while you're here

Reply:
I left it in because the tests here will be rewritten after the sugar is.